bgpd/bgp_vty.c

```diff
@@ -17234,6 +17234,9 @@ static void bgp_config_write_peer_af(struct vty *vty, struct bgp *bgp,
 					: "");
 		}
 	}
+
+	if (peer_af_flag_check(peer, afi, safi, PEER_FLAG_CONFIG_DAMPENING))
```

Comment:
shouldn't this be `peergroup_af_flag_check`?

Reply:
I currently believe that the dampening is not handling inheritance in the same way as the rest of the code.  As such I do not believe that that would work properly.  All this code is doing is just collapsing the multiple loops over the peer groups and peers into the one proper function.  So I am not changing functionality I am reducing looping.

Reply:
Sure.